src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+    public static final String LINE = "____________________________________________________________";
+
+    public static void printList(Task[] list) {
+        System.out.println(LINE);
+        if (list[0] == null) {
+            System.out.println("No items added!");
+        }
+        int i = 0;
+        while (list[i] != null) {
+            System.out.println((i+1) + ". " + list[i]);
+            i += 1;
+        }
+        System.out.println(LINE);
+    }
+
+    public static void markTaskAsDone(Task[] list, int taskIndex) {
+        if (taskIndex < Task.numItemsAdded && taskIndex >= 0) {
+            if (list[taskIndex].isDone()) {
+                System.out.println(LINE + "\nThat task is already done!\n" + LINE);
+            } else {
+                list[taskIndex].markAsDone();
+                System.out.println(LINE + "\nNice! I've marked this task as done:");
+                System.out.println(list[taskIndex] + "\n" + LINE);
+            }
+        } else {
+            System.out.println("That task does not exist!\n" + LINE);
+        }
+    }
+
+    public static void printAddedTask(Task[] list) {
+        System.out.println(LINE + "\nGot it. I've added this task:");
+        System.out.println(list[Task.numItemsAdded-1]);
+        System.out.format("Now you have %d tasks in the list.\n" + LINE + "\n", Task.numItemsAdded);
+    }
+
+    public static void addTask(Task[] list, String input, String[] inputWords) {
+        String taskType = inputWords[0];
+        if (taskType.equalsIgnoreCase("todo")) {
```

Comment:
Would it be better if you used switch statements instead, so that you could maybe cut down on the multiple layers of nesting?